### DISCUSSION TITLE
Can you share Firefox extensions across PWAs?

### DISCUSSION THREAD

**Discussion by keyframe-one** [filips123/PWAsForFirefox#336] [Q&A]
Firstly, thank you for this work. This 'profile' function has allowed me to separate work and personal accounts. 

I have created numerous (15+) PWAs and thus I had to install the Bitwarden password manager extension for every instance. 

Is it possible to share extensions across profiles?

**Comment by filips123** [comment]
You can try to use a normal Firefox Sync/Account to sync your settings and extensions.

You can also use profile templates to make installing the same set of extensions to new profiles easier. You can create one profile where you configure everything that you need. Then, you copy the profile directory somewhere else (so it's kept as a fresh template) and specify its path as "default profile template" in the extension settings. When you create a new profile, all files from that template will be copied to the new profile.

Continuous sync (other than Firefox Sync or third-party sync services) probably won't be implemented because it would be too complex.

**Comment by thedaveCA** [comment]
Another approach is to use Firefox `policies.json` to install a particular extension. The policy is placed in the runtime directory so care must be taken to avoid losing the file when the runtime is updated.
